docs/docsite/rst/dev_guide/developing_plugins.rst

```diff
@@ -63,18 +63,67 @@ To define configurable options for your plugin, describe them in the ``DOCUMENTA
         description: describe this config option
         default: default value for this config option
         env:
-          - name: NAME_OF_ENV_VAR
+          - name: MYCOLLECTION_NAME_OF_ENV_VAR
         ini:
-          - section: section_of_ansible.cfg_where_this_config_option_is_defined
-            key: key_used_in_ansible.cfg
+          - section: mycollection_section_of_ansible.cfg_where_this_config_option_is_defined
+            key: mycollection_key_used_in_ansible.cfg
```

Comment:
While the section should contain the collection's name somehow, the key doesn't have to repeat that info.
```suggestion
            key: key_used_in_ansible.cfg
```